src/main/python/janusgraph_python/structure/io/graphson/graphson_reader_builder.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONReader
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONUtil
+
+from ....serializer.relation_identifier_deserializer import RelationIdentifierDeserializer
+
+
+class JanusGraphSONReaderBuilder(object):
+    """
+    This class registers JanusGraph-specific deserializers so that objects like GeoShape, RelationIdentifier
```

Comment:
Drop "This class" and start with "Registers ..."

Replace "GeoShape, RelationIdentifier" with "GeoShape and RelationIdentifier"

Start the doc comment immediately after `"""`.

Reply:
AH okay. Didn't know that we don't add `space` after `"""`